torchtext/experimental/functional.py

```diff
@@ -1,4 +1,33 @@
 import torch
+from typing import List, Tuple
+
+
+def regex_tokenizer(line, patterns_list, regex, lowercase=False):
+    # type: (str, List[Tuple[str, str]], torch.classes.torchtext.Regex, bool) -> List[str]
+    r"""Regex tokenizer for a string sentence.
+
+    Steps include
+    - lowercasing (optional)
+    - applying all regex replacements defines in patterns_list
+
+    Args:
+        line (str): a line of text to tokenize.
+        patterns_list (List[Tuple[str, str]]): a list of tuples (ordered pairs) which contain the regex pattern string
+            as the first element and the replacement string as the second element.
+        regex (torch.classes.torchtext.Regex): a custom c++ Regex object.
+        lowercase (bool, optional): a flag to determine whether to lowercase the input text.
+    Returns:
+        tokens (List[str]): a list of tokens after splitting on whitespace.
+    """
+
+    if lowercase:
```

Comment:
Why do we need this as part of this?

Reply:
removed this function as part of our discussion